src/components/Staff/MinStaffWarning.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import {Alert, Box, Button} from "@mui/material";
+
+export interface IMinStaffWarning {
+  message1: string,
+  message2: string,
+  minStaff?: number,
+  handleClick: () => void,
+}
+
+export const MinStaffWarning = ({message1, message2, minStaff, handleClick}: IMinStaffWarning) => {
+  return <Alert sx={{color: '#873200', border: '1px solid #873200'}} severity="warning">
+    <Box sx={{fontWeight: 'bold'}}>{message1} {minStaff}</Box>
+    <Box sx={{paddingTop: '10px'}}>{message2}</Box>
+    <Box sx={{paddingTop: '10px'}}>
+      <Button
+        data-testid="mobile-add-min-staff"
+        variant="outlined"
+        id="add-min-staff"
+        onClick={handleClick}
+        sx={{marginLeft: 'auto', backgroundColor: '#334F96', color: '#fff', textTransform: 'none','&:hover': {backgroundColor: '#334F96'}}}
+      >+ Add minimum staff</Button>
```

Comment:
I'd probably use a button icon rather than a plus symbol here - you can pass one in using the startIcon prop:

`startIcon={<AddIcon />}`

https://mui.com/material-ui/react-button/#buttons-with-icons-and-label